tox.ini

```diff
@@ -1,16 +1,15 @@
 # Configuration of py.test
 [pytest]
 addopts=-v
+        -s
```

Comment:
What's the rationale for switching the file name?

Reply:
pycodestyle explicitly needs a setup.cfg (depreciated) or a tox.ini for adding any settings (like max line length), and pytest will gladly take either (we could also have a pytest.ini __and__ a tox.ini, but I figured they might as well be in the same file)

also, the added `-s` flag shows the rationale for skipped warnings.  I added it because pycodestyle will skip unchanged, previously passed files for style-review (it's safe in general, but can be confusing for local testing w/o the `-s` flag active.  tbh, I'm not a big fan of it, but there's no (well-documented) way to turn it off)